pkg/datapath/iptables/iptables.go

```diff
@@ -940,6 +940,12 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 	}
 
 	for _, c := range ciliumChains {
+		// do not install feeder for tables that are set to be disabled
+		for _, disabledTables := range option.Config.DisableIptablesFeedererRules {
+			if strings.ToLower(c.hook) == strings.ToLower(disabledTables) {
+				continue
```

Comment:
I think it'd be useful here to log at Info level to make it clear to the user that something was skipped here. This will be useful if it's misconfigured, so we know that it's not disabling the rules; and conversely if someone enables it and hits some weird issue, we can also see evidence in the logs.

Reply:
Done :smile: